dallinger/default_configs/local_config_defaults.txt

```diff
@@ -0,0 +1,35 @@
+[HIT Configuration]
+title = Stroop task
+description = Judge the color of a series of words.
+amt_keywords = Perception, Psychology
+lifetime = 24
+us_only = true
+approve_requirement = 95
+contact_email_on_error = youremail@gmail.com
+ad_group = Default psiTurk Stroop Example
+psiturk_keywords = stroop
+organization_name = New Great University
+browser_exclude_rule = MSIE, mobile, tablet
+
+[Database Parameters]
+database_url = sqlite:///participants.db
+table_name = turkdemo
+
+[Server Parameters]
+host = localhost
+port = 22362
+cutoff_time = 30
+logfile = server.log
+loglevel = 2
+debug = true
+login_username = examplename
+login_pw = examplepassword
+threads = auto
+
+[Task Parameters]
+experiment_code_version = 1.0
+num_conds = 1
+num_counters = 1
+
+[Shell Parameters]
+launch_in_sandbox_mode = true
```

Comment:
Maybe we should copy the defaults present in one of the Dallinger demos rather than the psiTurk defaults because we have some parameters they don't, and vice versa.